test/tests/functional/pbs_node_rampdown.py

```diff
@@ -429,7 +438,7 @@ def setUp(self):
             "#PBS -S /bin/bash\n" \
             "#PBS -l select=" + self.job1_select + "\n" + \
             "#PBS -l place=" + self.job1_place + "\n" + \
-            SLEEP_CMD + " 10\n" + \
+            SLEEP_CMD + " 5\n" + \
```

Comment:
Don't need to sleep too long before the pbs_release_nodes call, for test to capture the assigned values before the release call.